test/policies/exponentialRetryPolicyTests.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+import * as assert from "assert";
+import { HttpMethod } from "../../lib/httpMethod";
+import { HttpRequest } from "../../lib/httpRequest";
+import { HttpResponse } from "../../lib/httpResponse";
+import { InMemoryHttpResponse } from "../../lib/inMemoryHttpResponse";
+import { exponentialRetryPolicy } from "../../lib/policies/exponentialRetryPolicy";
+import { RequestPolicy } from "../../lib/requestPolicy";
+import { RequestPolicyFactory } from "../../lib/requestPolicyFactory";
+import { RequestPolicyOptions } from "../../lib/requestPolicyOptions";
+
+describe("exponentialRetryPolicy", () => {
```

Comment:
General comment: It may be fine for now, but we also need to revisit mechanism to record tests. We have been using a super old version of nock.js for test recording. Now since we are rewriting stuff we should take a fresh stab at it.